### ISSUE TITLE
topbar scss colors

### ISSUE THREAD

**Issue by richpeers** [zurb/foundation-sites#7900]
via bower ( 6.1.1 ) , and via node cli, also via zip download of develop branch,
editing _settings.scss like:
```
$topbar-background: $black;
$topbar-link-color: $white;
```
app.scss
```
@import 'settings';
@import 'foundation';
@include foundation-everything($flex: true);
```
and compiles fine using node cli, no errors

the $topbar-background does not cascade down menus within the top bar
the text (title) in the topbar is still black
the $topbar-link-color does apply, but the down arrows are still default $primary-color

![untitled](https://cloud.githubusercontent.com/assets/2292987/12378358/d7043ee6-bd32-11e5-8c9b-a8c190c4136f.png)

**Comment by gakimball** [comment]
Here's the CSS that applies the background color. It does it to the top bar itself, and every `<ul>` inside.

```css
.top-bar,
  .top-bar ul {
    background-color: #e6e6e6; }
```

Try inspecting the dropdown and see if another style is overriding this rule.

**Comment by richpeers** [comment]
It's overridden by the menu classes, like
ul.menu 
Don't remember the exact class, but it was the menu stuff

**Comment by richpeers** [comment]
Menu scss stuff came out in the css before the topbar scss as its ordered in source files,  but the menu stuff is just more specific

**Comment by gakimball** [comment]
Cut back the specificity of a menu selector to fix the issue: 0ce09db0424dae44868ae5e0b5fc2218bcfd51e2
